driver-scala/src/main/scala/org/mongodb/scala/FindObservable.scala

```diff
@@ -310,5 +311,31 @@ case class FindObservable[TResult](private val wrapped: FindPublisher[TResult])
     this
   }
 
+  /**
+   * Explain the execution plan for this operation with the server's default verbosity level
+   *
+   * @return the execution plan
+   * @since 4.2
+   * @note Requires MongoDB 3.2 or greater
+   */
+  def explain[ExplainResult]()(
```

Comment:
I'm not too confident about the Scala I've written.

Reply:
Looks good just missing the `@tparam` in the scala doc.

Reply:
Uh, what `@tparam`?  Can you indicate the exact change required?

Reply:
Its the documentation for the type so something like:

```
/**
   * Explain the execution plan for this operation with the server's default verbosity level
   *
   * @tparam ExplainResult The type of the result 
   * @return the execution plan
   * @since 4.2
   * @note Requires MongoDB 3.2 or greater
   */
  def explain[ExplainResult]()(
```

Reply:
Done